packages/components/src/components/icon/common/ic-linux.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 6C0 2.68629 2.68629 0 6 0H34C37.3137 0 40 2.68629 40 6V34C40 37.3137 37.3137 40 34 40H6C2.68629 40 0 37.3137 0 34V6Z" fill="#F2F3F4"/>
```

Comment:
can we use css var in fill?
`fill="var(--general-section-1)"` instead of `fill="#F2F3F4"`